tools/CreateMappings.ps1

```diff
@@ -57,5 +57,9 @@ $results | ConvertTo-Json | Out-File $OutputFile;
 # Print conclusion.
 Write-Host ""
 Write-Host "$($results.Count) cmdlets successfully mapped: $($OutputFile)." -ForegroundColor Green;
-Write-Host "$($warnings.Count) cmdlets could not be mapped and were placed in 'Other': $($WarningFile)." -ForegroundColor Yellow;
 Write-Host ""
+
+if($warnings.Count -gt 0) {
+    Write-Host "$($warnings.Count) cmdlets could not be mapped and were placed in 'Other': $($WarningFile)." -ForegroundColor Yellow;
+    throw "Some cmdlets could not be properly mapped to a documentation grouping: $($warnings -join ", ").  Please add a mapping rule to $(Resolve-Path -Path $RulesFile).";
```

Comment:
@twitchax I guess one more comment: if an on-boarding service breaks their PR's build due to this failure, do we have any guidance around what their `regex`/`alias` keys should map to in the `.json` file?

Reply:
Not yet: we should probably add something to the developer guide.  However, looking at the file makes it fairly self-explanatory.